x/name/keeper/keeper_test.go

```diff
@@ -638,7 +479,7 @@ func (s *KeeperTestSuite) TestCreateRootNameProposals() {
 			name:       "..badroot",
 			owner:      s.user1Addr,
 			restricted: false,
-			err:        fmt.Errorf("segment of name is too short"),
+			err:        fmt.Errorf("name is already bound to an address"),
 		},
```

Comment:
issue: This doesn't make any sense to me. How can that be the expected error when the name should never be bound in the first place?